nikola/plugin_categories.py

```diff
@@ -303,6 +303,11 @@ def compile(self, source, dest, is_two_file=True, post=None, lang=None):
         # ignore `compile_html`
         self.compile_html(source, dest, is_two_file)
 
+    def compile_string(self, data, source_path=None, is_two_file=True, post=None, lang=None):
+        """Compile the source file into HTML strings (with shortcode support)."""
```

Comment:
Maybe the documentation should say that this has to return a tuple whose first component is the HTML fragment?